operations/services.md

```diff
@@ -214,6 +216,134 @@ Updates the number of available resources in [Resource category](enterprises.md#
+            "StartUtc": "2020-11-05T00:00:00Z",
+            "EndUtc": "2020-11-06T00:00:00Z",
+            "ReleasedUtc": "2020-11-04T00:00:00Z",
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `AvailabilityBlocks` | array of [Availability block parameters](#availability-block-parameters) | required, max 1000 items | Availability blocks to be added. |
+
+#### Availability block parameters
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `ServiceId` | string | required | Unique identifier of the [Service](#service) to assign block to. |
+| `StartUtc` | string | required | Start of the interval in UTC timezone in ISO 8601 format. |
+| `EndUtc` | string | required | End of the interval in UTC timezone in ISO 8601 format. |
+| `ReleasedUtc` | string | required | Date when the [Availability update](#availability-update)s linked to the block will not reduce public availability in UTC timezone in ISO 8601 format. |
+
+### Response
+
+```javascript
+{
+    "AvailabilityBlocks": [
+        {
+            "Id": "e5a4654a4a94-86da-4f96-9efc-bd26d8db",
+            "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
+            "StartUtc": "2020-10-05T00:00:00Z",
+            "EndUtc": "2020-10-06T00:00:00Z",
+            "ReleasedUtc": "2020-10-04T00:00:00Z",
+        },
+        {
+            "Id": "aaaa654a4a94-4f96-9efc-86da-bd26d8db",
+            "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
+            "StartUtc": "2020-11-05T00:00:00Z",
+            "EndUtc": "2020-11-06T00:00:00Z",
+            "ReleasedUtc": "2020-11-04T00:00:00Z",
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `AvailabilityBlocks` | array of [Availability blocks](#availability-block) | required | Availability blocks. |
+
+#### Availability block
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `Id` | string | required | Unique identifier of the availability block. |
+| `ServiceId` | string | required | Unique identifier of the [Service](#service) the block is assigned to. |
+| `StartUtc` | string | required | Start of the interval in UTC timezone in ISO 8601 format. |
+| `EndUtc` | string | required | End of the interval in UTC timezone in ISO 8601 format. |
+| `ReleasedUtc` | string | required | Date when the [Availability update](#availability-update)s linked to the block will not reduce public availability in UTC timezone in ISO 8601 format. |
+
+## Delete availability blocks
+
+Delete availability blocks. Only block with no active [Availability update](#availability-update)s and [Reservation](#reservation)s can be deleted
```

Comment:
what does "Only block with no active [Availability update](#availability-update)s" mean?

Reply:
There cant be allocated any availability. -> they have to setup availability for all categories within block to null

Reply:
ah ok got it. I think then that might need to be specified. Or we can leave it as is, and I'll suggest an update if someone starts asking. 

Would the reverse work? Can you adjust availability on dates that already have an Availability block set?

Reply:
It sounded to me logical.. we can remove it